modules/http/http-worker.c

```diff
@@ -345,6 +402,59 @@ _debug_response_info(HTTPDestinationWorker *self, HTTPLoadBalancerTarget *target
             log_pipe_location_tag(&owner->super.super.super.super));
 }
 
+static LogThreadedResult
+_custom_map_http_result(HTTPDestinationWorker *self, const gchar *url, HttpResponseHandler *response_handler)
+{
+  HttpResult result = response_handler->action(response_handler->user_data);
+  g_assert(result < HTTP_RESULT_MAX);
+  HTTPDestinationDriver *owner = (HTTPDestinationDriver *) self->super.owner;
+  glong http_code = response_handler->status_code;
+
+  switch (result)
+    {
+    case HTTP_RESULT_SUCCESS:
+      msg_debug("http: handled by on_error",
+                evt_tag_str("decision", "success"),
```

Comment:
decision is probably better named "action", like in the naming of the grammar rules.

Reply:
fixed